src/redux/appReducer.js

```diff
@@ -0,0 +1,23 @@
+import * as buttonLogic from '../pressButtonLogic';
+import * as actions from './actionTypes';
+export default function appReducer(
+    state = {
+        result: ['0'],
+        operators: [],
+        redundant: 0,
+    },
+    action
+) {
+    const { type, value, displayable } = { ...action };
```

Comment:
We talked about this `displayable` variable and it seemed that we agreed that you change your code so that you don't use it in the action. Did I miss anything?